### PR TITLE
GOB: added new variants of Adiboud'chou to detection tables

### PR DESCRIPTION
**Pull request by BJNFNE**

### CODE REVIEW

**Comment by sdelamarre** [comment]
Thank you!

File: engines/gob/detection/tables_adiboudchou.h [hunk 1]
```diff
@@ -58,5 +58,35 @@
 	kFeatures640x480,
 	0,0,0
 },
+{
+
```
Comment (sdelamarre) [inline engines/gob/detection/tables_adiboudchou.h:62]: Useless blank line

File: engines/gob/detection/tables_adiboudchou.h [hunk 2]
```diff
@@ -58,5 +58,35 @@
...
+		ADGF_UNSUPPORTED,
+		GUIO0()
+	},
+	kGameTypeNone,
+	kFeatures640x480,
+	0,0,0
+},
+{
+	{
+
```
Comment (sdelamarre) [inline engines/gob/detection/tables_adiboudchou.h:78]: Useless blank line

**Review by sdelamarre — CHANGES_REQUESTED** [review]

**Comment by sdelamarre** [comment]
Can you remove the two spotted useless blank lines, and also the one at line 48 (adiboudchoubanquise) introduced in your last commit?

**Comment by BJNFNE** [comment]
I think i messed this one up, i am no longer able to merge / squash the commits anymore, i begin from scratch with a new PR

**Comment by sdelamarre** [comment]
Squashed, thank you.

**Pull request merged**